### DISCUSSION TITLE
Welcome to matterbridge Discussions!

### DISCUSSION THREAD

**Discussion by 42wim** [42wim/matterbridge#1314] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. 
Feel free to:
  * Ask [questions](https://github.com/42wim/matterbridge/discussions?discussions_q=category%3AQ%26A).
  * Share [ideas](https://github.com/42wim/matterbridge/discussions?discussions_q=category%3AIdeas), cool tricks or tengo scripts.
  * [Engage](https://github.com/42wim/matterbridge/discussions?discussions_q=category%3A%22Show+and+tell%22) with other community members.
 
Welcome everyone!

For real bugs please use the [issues](https://github.com/42wim/matterbridge/issues)

**Comment by qaisjp** [comment]
Amazing! How did you turn this on? I've been waiting for this for AGES.

Edit: [found you :)](https://github.community/t/can-one-apply-for-github-discussions-beta/121120/471?u=qaisjp)

**Comment by otnooishphoo** [comment]
i am not a dev, but i wonder if an echomail layer is possible. to sync to old school ssh/telnet BBS message boards.
